src/Bridges/bridgeoptimizer.jl

```diff
@@ -318,11 +320,36 @@ end
 # Name
 function MOI.canget(b::AbstractBridgeOptimizer, IdxT::Type{<:MOI.Index},
                     name::String)
-    return MOI.canget(b.model, IdxT, name)
+    if isbridged(b, IdxT)
+        return MOI.canget(b.bridged, IdxT, name)
+    else
+        return MOI.canget(b.model, IdxT, name)
+    end
 end
 function MOI.get(b::AbstractBridgeOptimizer, IdxT::Type{<:MOI.Index},
                  name::String)
-    return MOI.get(b.model, IdxT, name)
+    if isbridged(b, IdxT)
+        return MOI.get(b.bridged, IdxT, name)
+    else
+        return MOI.get(b.model, IdxT, name)
+    end
+end
+
+# For the type-unstable case, we have no information as to whether the
+# constraint is in the bridge or the model. Therefore, we try the model first,
+# and then the bridge if that fails. As such, we need canget for either the
+# bridge or the model.
+function MOI.canget(b::AbstractBridgeOptimizer, IdxT::Type{CI},
+                    name::String)
+    return MOI.canget(b.bridged, IdxT, name) || MOI.canget(b.model, IdxT, name)
+end
+function MOI.get(b::AbstractBridgeOptimizer, IdxT::Type{CI},
+                 name::String)
+    try
```

Comment:
We should use `canget(b.model, IdxT, name)` instead of `try` `catch` as done in https://github.com/JuliaOpt/MathOptInterface.jl/blob/c164eabcff67e6d96341e69eb053e6fc01b5e3a3/src/Utilities/universalfallback.jl#L229-L235
This need for `canget` is talked about in https://github.com/JuliaOpt/MathOptInterface.jl/issues/424.

Reply:
We'll replace it with `try` ... `catch` when we remove `canget` at both places if that is the solution we choose.